packages/web-forms/src/components/common/map/MapBlock.vue

```diff
@@ -266,22 +267,22 @@ const discardSavedFeature = () => {
 	right: var(--odk-map-spacing);
 	bottom: 35px;
 	display: flex;
+	gap: 9px;
```

Comment:
Surely this should match the `.control-bar`?


```suggestion
	gap: 8px;
```